static/styles/portico/stats.scss

```diff
@@ -29,45 +29,43 @@ p {
 .chart-container {
     display: inline-block;
     vertical-align: top;
-
     margin: 10px 0px;
     padding: 20px;
-
     border: 2px solid hsl(0, 0%, 93%);
     background-color: hsl(0, 0%, 100%);
-}
 
-.chart-container button {
-    position: relative;
-    z-index: 1;
-}
+    button {
+        position: relative;
+        z-index: 1;
+    }
 
-.chart-container h1 {
-    margin-top: 0px;
-    font-size: 1.5em;
-}
+    h1 {
+        margin-top: 0px;
+        font-size: 1.5em;
+    }
 
-.chart-container:not(.pie-chart) .number-stat {
-    position: relative;
-    top: -30px;
-}
+    .button-container {
+        position: relative;
+        z-index: 1;
 
-.chart-container.pie-chart hr {
-    margin-bottom: 8px;
-}
+        label {
+            margin: 3px;
+        }
 
-.chart-container .button-container {
-    position: relative;
-    z-index: 1;
-}
+        > * {
+            display: inline-block;
+            vertical-align: top;
+        }
+    }
 
-.chart-container .button-container label {
-    margin: 3px;
-}
+    &:not(.pie-chart) .number-stat {
+        position: relative;
+        top: -30px;
+    }
 
-.chart-container .button-container > * {
-    display: inline-block;
-    vertical-align: top;
+    &.pie-chart hr {
+        margin-bottom: 8px;
+    }
```

Comment:
This commit is pretty hard to read with the reordering mixed with changing indentation; can you split it to reorder first?

Reply:
Fixed this in the latest push! :smile: